### PR TITLE
Updated and corrected the documentation 

### PR DESCRIPTION
**Pull request by AshutoshMourya**

### CODE REVIEW

**Review by abhishekpatel946 — CHANGES_REQUESTED** [review]
Can you pls remove the unusual comments from readme.

And make sure that if you change something in the code write down the proper commit mgs as per convention. Short % Informatic.

File: README.md [hunk 1]
```diff
@@ -11,45 +11,54 @@
...
 
 
 
+
 <!-- ABOUT THE PROJECT -->
 ## About The Project
 <p align="center">
   <a href="https://github.com/abhishekpatel946/instagram-clone">
-<!--     <img src="Icon/logo-passmanager.jpeg" alt="Logo" width="100" height="100"> -->
+<!--     <img src="Icon/logo-passmanager.jpeg" alt="Logo" width="10" height="10"> -->
```
Comment (abhishekpatel946) [inline README.md]: remove this line of code which already comments.

**Review by abhishekpatel946 — CHANGES_REQUESTED** [review]
remove those line which I highlighted

File: README.md (hunk 1 above)
Comment (AshutoshMourya) [reply README.md]: ok

**Pull request merged**